Dockerfile.plainbundle

```diff
@@ -0,0 +1,3 @@
+FROM scratch
+LABEL manifests="/manifests"
```

Comment:
nit: I would avoid adding labels until we have some kind of consensus around metadata and whether the labels are helpful. Right now we have the limitation that for a given plain+v0 bundle image to be valid it must have a `/manifests` directory at the root, so this label is redundant.